Count/ViewController.swift

```diff
@@ -10,47 +10,65 @@ import UIKit
 
 class ViewController: UIViewController {
     
-    var number: Int = 0
     @IBOutlet var label: UILabel!
-
+    @IBOutlet var bestView: BestLabel!
+    private var timer: Timer?
+    private var timerTotalDuration: TimeInterval = 0
+    var db = UserDefaults.standard
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        db.register(defaults: ["highestScore":0.0])
+        bestView.text = String(format: "%.2f",db.double(forKey: "highestScore"))
     }
-
+    
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
     
-    @IBAction func plus() {
-        
-        number = number + 3
-        label.text = String(number)
+    @IBAction func startBtn(){
```

Comment:
```suggestion
    @IBAction func start(){
```

もしくは

```suggestion
    @IBAction func tappedStartButton(){
```

とかにしたほうが命名としては理想かも。伊豆でレビューしていくこと考えるとなるべく省略形は使わないのが理想